src/applications/mhv-inherited-proofing/constants.js

```diff
@@ -0,0 +1,52 @@
+import { signup } from 'platform/user/authentication/utilities';
+import { CSP_IDS } from 'platform/user/authentication/constants';
+import recordEvent from 'platform/monitoring/record-event';
+import environment from 'platform/utilities/environment';
+
+export const TRANSITION_ENDPOINT = `${
+  environment.API_URL
+}/inherited_proofing/auth`;
+
+export const EVENTS = {
+  TRANSITION_STARTED: 'login-transition-started-mhv-to-logingov',
+  TRANSITION_PAGE_DISMISSED: 'login-transition-page-dismissed',
+};
+
+export const ACCOUNT_TRANSITION = {
+  headline: {
+    eligible: 'Transfer your account to Login.gov now',
+    ineligible: 'Create your new account',
+  },
+  subheader: {
+    eligible: 'This process should take about [X] minutes.',
```

Comment:
Create a ticket as a reminder to resolve `[X]` this so it does not fall through the cracks and get released

Reply:
Let me abstract it out to a constant that way it will be a simple fix like the date.